website/cue/reference/remap/functions/parse_groks.cue

```diff
@@ -33,9 +33,18 @@ remap: functions: parse_groks: {
 			default:     true
 			type: ["object"]
 		},
+		{
+			name:        "alias_sources"
+			description: "Path to file containing aliases in a JSON format."
```

Comment:
Will it be clear to users what the schema of the JSON objects should be? If not, could we include an example?